activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb

```diff
@@ -171,6 +164,13 @@ def translate_exception(exception, message:, sql:, binds:)
             super
           end
         end
+
+        def default_prepared_statements # :nodoc:
+          ActiveSupport::Deprecation.warn(<<-MSG.squish)
+          The default value of `prepared_statements` for the mysql2 adapter will be changed from +false+ to +true+ in Rails 7.2.
```

Comment:
```suggestion
            The default value of `prepared_statements` for the mysql2 adapter will be changed from +false+ to +true+ in Rails 7.2.
```

Just matching the indentation from the previous method.